web_server/friendship/migrations/0001_initial.py

```diff
@@ -11,15 +11,37 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='FriendRequest',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('from_id', models.CharField(max_length=500)),
+                ('to_id', models.CharField(max_length=500)),
+            ],
+            options={
+                'unique_together': {('from_id', 'to_id')},
+            },
+        ),
         migrations.CreateModel(
             name='Friend',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('author_id', models.URLField(max_length=500)),
-                ('friend_id', models.URLField(max_length=500)),
+                ('author_id', models.CharField(max_length=500)),
+                ('friend_id', models.CharField(max_length=500)),
```

Comment:
Something is off here, why are you changing the friendship models to implement the github feature? Did you accidentally make migrations on old model files?

Reply:
I am not actually sure what is up with the migrations. I had deleted everything and re migrated, as I was having DB conflicts before. Maybe master has some weird migration?